### PR TITLE
[RFR] Added “Vanilla” entry

### PR DESCRIPTION
**Pull request by FrankDraws**
This is my first official contribution to GitHub (i.e. pull-request). If I made any mistake, please let me know so I can learn from it.

I added a definition for the VANILLA.md file as well as edited the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 ### V
 
 * [V8](/glossary/V8.md): Google’s open source javaScript engine. It’s what Chrome is running, but it’s also used for other projects like Node.js and MongoDB.
-* [Vanilla](/glossary/VANILLA.md): *to be completed.*
+* [Vanilla](/glossary/VANILLA.md): Plain JavaScript.
```
Comment (KittyGiraudel) [inline README.md:169]: What about something a bit more verbose such as “a term used to describe plain JavaScript, with no library and/or framework”?

**Comment by KittyGiraudel** [comment]
Hey. Thank you very much for your contribution. Super glad to have your first. :)

File: README.md (hunk 1 above)
Comment (FrankDraws) [inline README.md:169]: Will do. I'll revise the entry and resubmit.

Thanks for the feed back!

Comment (FrankDraws) [inline README.md:169]: How about the following?

"A term utilized to describe the use of plain JavaScript without dependencies such as libraries and/or frameworks."

**Pull request merged**